RubberduckTests/Inspections/AssignedByValParameterInspectionTests.cs

```diff
@@ -170,6 +170,40 @@ End Sub
             }
         }
 
+        [Test]
+        [Category("Inspections")]
+        public void AssignedByValParameter_NoResultForDefaultMembberAssignment()
+        {
+            var class1 = @"
+Public Property Get Something() As Long
+Attribute Foo.VB_UserMemId = 0
+End Property
+Public Property Let Something(ByVal value As Long)
+Attribute Foo.VB_UserMemId = 0
```

Comment:
Not that it makes a difference, but is this legal code? IIUC only one member is allowed to have `VB_UserMemId = 0`... I've always only ever put it on the `Property Get` member.

Reply:
I have no idea. I guess the VBE would just ignore  one of them.